packages/shared/src/broker.ts

```diff
@@ -206,15 +206,15 @@ export async function requestSwapDepositAddress(
   const params = getDepositAddressRequestSchema(chainflipNetwork).parse(request);
 
   const response = await client.sendRequest(
-    'broker_requestSwapDepositAddress',
+    'broker_request_swap_deposit_address',
     params.srcAsset,
     params.destAsset,
     params.destAddress,
     params.commissionBps,
     params.ccmParams,
     params.maxBoostFeeBps,
     params.affiliates,
-    params.fillOrKillParams,
+    params.fillOrKillParams!,
```

Comment:
wanted to keep this pr related to the package update. will property adjust types in a follow up together with validation + changelog